src/components/repository-list-item.component.js

```diff
@@ -80,7 +81,9 @@ const renderTitle = repository =>
       />
 
       <Text style={[styles.extraInfoSubject, { paddingTop: 2 }]}>
-        {repository.stargazers_count}
+        {repository.stargazers_count >= 1000
```

Comment:
Did you decide to not have this in `utils`? I think it might actually be better the way you had it before (separate method in `utils` for `getStartNumber` for example. This way we can import the method and use it here _and_ in `repository-profile.component.js` where the stars show on the repository screen.

Let me know if that's alright with you!